libwaysip/examples/base.rs

```diff
@@ -20,11 +20,28 @@ impl Dispatch<wl_registry::WlRegistry, GlobalListContents> for State {
 fn main() {
     let connection = Connection::connect_to_env().unwrap();
 
+    let color = libwaysip::state::Color {
+        r: 0.0,
+        g: 0.0,
+        b: 0.0,
+        a: 1.0,
+    };
+
+    let passing_data = libwaysip::state::PassingData {
```

Comment:
I think maybe we can without the PassingData, just add field

background_color, foreground_color, border_text_color, fond_size to the Waysip field, and add 6 function to build the waysip.